src/Reshuffle.ts

```diff
@@ -105,8 +104,22 @@ export default class Reshuffle {
     Object.values(this.registry.connectors).forEach((connector) => connector.start())
 
     // Start the webserver if we have http delegates
-    if (this.registry.common.webserver) {
-      this.httpServer = this.registry.common.webserver.listen(this.port, () => {
+    if (Object.keys(this.httpDelegates).length) {
+      const webserver = this.prepareWebServer()
```

Comment:
I like how now we only have a webserver if there are HTTP delegates 👍

Reply:
can't register new routes after start() but I don't think it ever works